unlock-js/src/walletService.js

```diff
@@ -88,17 +89,18 @@ export default class WalletService extends UnlockService {
 
     if (!accounts.length) {
       // We do not have an account, can't do anything until we have one.
-      return (this.ready = false)
+      this.ready = false
```

Comment:
This is a lint change, only picked up in the pre-commit hook for whatever reason. It doesn't like assigning in a return statement (and I agree)